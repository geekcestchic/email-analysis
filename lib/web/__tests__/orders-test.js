import Orders from '../components/orders'
import renderer from 'react-test-renderer'
import order from '../test-data/order-object'
import React from 'react'

let orders = [order, order, order, order]

it('renders correctly', () => {
  const component = renderer.create(
    <Orders orders={orders}/>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree.children.length).toBe(4 + 1) // The title!
})
