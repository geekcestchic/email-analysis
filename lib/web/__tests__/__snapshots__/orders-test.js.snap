exports[`test renders correctly 1`] = `
<div
  className="orders"
  style={
    Object {
      "textAlign": "center"
    }
  }>
  <h1
    className="orders-title"
    style={
      Object {
        "borderBottom": "3px dotted black",
        "fontFamily": "arial",
        "padding": "20px 0px 30px",
        "textAlign": "center"
      }
    }>
    Orders Dashboard
  </h1>
  <div
    className="order"
    onClick={[Function bound onClick]}
    style={
      Object {
        "border": "1px solid black",
        "borderRadius": 10,
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "arial",
        "margin": "2.5%",
        "padding": 20,
        "textAlign": "center",
        "width": "40%"
      }
    }>
    <div>
      Order For - 
      Adham Bard
    </div>
    <div>
      Payment Status: 
      Paid 3748 EUR
    </div>
    <button
      style={
        Object {
          "background": "black",
          "border": "none",
          "borderRadius": 0,
          "color": "white",
          "marginTop": 5,
          "padding": "5px 10px"
        }
      }>
      Click for detail
    </button>
  </div>
  <div
    className="order"
    onClick={[Function bound onClick]}
    style={
      Object {
        "border": "1px solid black",
        "borderRadius": 10,
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "arial",
        "margin": "2.5%",
        "padding": 20,
        "textAlign": "center",
        "width": "40%"
      }
    }>
    <div>
      Order For - 
      Adham Bard
    </div>
    <div>
      Payment Status: 
      Paid 3748 EUR
    </div>
    <button
      style={
        Object {
          "background": "black",
          "border": "none",
          "borderRadius": 0,
          "color": "white",
          "marginTop": 5,
          "padding": "5px 10px"
        }
      }>
      Click for detail
    </button>
  </div>
  <div
    className="order"
    onClick={[Function bound onClick]}
    style={
      Object {
        "border": "1px solid black",
        "borderRadius": 10,
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "arial",
        "margin": "2.5%",
        "padding": 20,
        "textAlign": "center",
        "width": "40%"
      }
    }>
    <div>
      Order For - 
      Adham Bard
    </div>
    <div>
      Payment Status: 
      Paid 3748 EUR
    </div>
    <button
      style={
        Object {
          "background": "black",
          "border": "none",
          "borderRadius": 0,
          "color": "white",
          "marginTop": 5,
          "padding": "5px 10px"
        }
      }>
      Click for detail
    </button>
  </div>
  <div
    className="order"
    onClick={[Function bound onClick]}
    style={
      Object {
        "border": "1px solid black",
        "borderRadius": 10,
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "arial",
        "margin": "2.5%",
        "padding": 20,
        "textAlign": "center",
        "width": "40%"
      }
    }>
    <div>
      Order For - 
      Adham Bard
    </div>
    <div>
      Payment Status: 
      Paid 3748 EUR
    </div>
    <button
      style={
        Object {
          "background": "black",
          "border": "none",
          "borderRadius": 0,
          "color": "white",
          "marginTop": 5,
          "padding": "5px 10px"
        }
      }>
      Click for detail
    </button>
  </div>
</div>
`;
